### DISCUSSION TITLE
AR 개발 참고자료

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/UnityAR#1] [General]
1. Unity 공식 Tutorial 
https://learn.unity.com/project/get-started-with-ar
https://learn.unity.com/project/create-a-marker-based-ar-app?language=en

2. 3D 객체를 손 위에 올려서 조작하는 영상 (25분 35초) 
https://www.youtube.com/watch?v=gpaq5bAjya8

3. Dilmer Valecillos 유튜브 채널 (자료 엄청 많음)
https://www.youtube.com/watch?v=cfKzUYH4i7A&list=PLQMQNmwN3FvzLN-8moCKmZb00gr7sdcrZ&index=1

4. 순순 스튜디오 한글 튜토리얼
https://www.youtube.com/playlist?list=PLATff1khpIscTMN2KXxagrTkBCwdzOeol

**Comment by cheona-thousand-man** [comment]
5. Niantic LightShip AR 유투브 채널
https://www.youtube.com/@LightshipAR

6. Dilmer Valecillos 유튜브 채널에서 LightShip AR 부분만 ← Lightship2 버전이라 지금 버전(Lightship3)과 좀 다름
https://www.youtube.com/playlist?list=PLQMQNmwN3FvxnT4KRXdYPGhPZ2kNWuOij

**Comment by cheona-thousand-man** [comment]
7. LightShip 공식 개발 기초 가이드
https://lightship.dev/docs/ardk/

**Comment by cheona-thousand-man** [comment]
8. Alive studios 유투브 채널(Lightship 3.0 튜토리얼)
https://www.youtube.com/watch?v=7Eopoo3QWPQ&list=PL5g7DLqxoknizynWvzBaz1HQ_HunIf97d
